src/openrct2/rct1.h

```diff
@@ -721,8 +722,8 @@ typedef struct rct1_s4 {
     rct_ride_measurement ride_measurements[8];
     uint32 next_guest_index;
     uint16 game_counter_5;
-    uint8 patrol_areas[0x3C00];
-    uint8 unk_1F42AA[116];
+    uint8 patrol_areas[(RCT1_MAX_STAFF + 4) * 128];
```

Comment:
Why `+4`? And what is the meaning of 128?

Reply:
For every staff type, there is an additional "meta-patrol-area", which is the union of all patrol areas for that type. E.g. there is one combined patrol area of all janitors, one of all mechanics, etc. There are four staff types, that's why +4.
Every patrol area needs to be able to cover the entire park, and you need 128 uint8s for this (that's 1024 bits, and one bit represents one of the 4x4 squares. Max map size is 128x128, which is 32x32=1024 4x4 squares)
Should this be documented somehow? The RCT2 part is basically the same, but with a bigger map size, and isn't really documented either.

Reply:
Yes please document that as well. We are trying to get rid of as many magic numbers as possible